lab-bundle-spec.md

```diff
@@ -20,6 +20,8 @@ The primary changes made from v1 to v2 are:
 
 - Adding an `ssh_key_user` attribute on the `gcp_project` resource type.
 
+- Adding a `console_url` reference to the `gcp_project` resource type.
```

Comment:
I'd probably say something more broadly like "Requiring lab creators to specify exact control panel outputs with `student_visible_outputs`."

Reply:
updated